src/mexican_wave.py

```diff
@@ -0,0 +1,15 @@
+def wave(string):
+    string.casefold()
```

Comment:
I see you added this line.  The challenge says the input will be lower case, so I'm not sure why it is needed.  If you really want it, write a test to prove it is doing what you expect.

Reply:
okay, thank you, I will add a test to prove that it works.
codeKata link:
https://www.codewars.com/kata/58f5c63f1e26ecda7e000029/train/python